api/app/src/main/resources/application-test.properties

```diff
@@ -1,7 +1,8 @@
-spring.datasource.driver-class-name=org.h2.Driver
-spring.datasource.url=jdbc:h2:mem:test
-spring.datasource.username=sa
-spring.datasource.password=
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.H2Dialect
```

Comment:
Using a fake db server and a different SQL dialect was resulting in bugs not being surfaced in the tests. There's no need to write unit tests against repositories because they don't contain any logic - all we care about is whether they integrate correctly with a Postgres database.